rootToH5converter/Event.py

```diff
@@ -116,16 +133,19 @@ def fill_jets(self, use_fat_jets=False):
                       n_neutral=self.data_processor.get_value_from_tree(prefix+"Jet_nNeutral", self.i_event, i_jet),
                       ch_hef=self.data_processor.get_value_from_tree(prefix+"Jet_chHEF", self.i_event, i_jet),
                       ne_hef=self.data_processor.get_value_from_tree(prefix+"Jet_neHEF", self.i_event, i_jet))
-            
+
             # check if tree contains links between tracks and jets
             jet_index = -1
             track_jet_index = None
-            if self.data_processor.get_value_from_tree("Track_jet_index") is not None:
-                track_jet_index = self.data_processor.get_value_from_tree("Track_jet_index", self.i_event)
+            track_cand_index = None
+            if self.data_processor.get_value_from_tree("Track_jet_index_"+AKx) is not None:
+                track_jet_index = self.data_processor.get_value_from_tree("Track_jet_index_"+AKx, self.i_event)
                 jet_index = i_jet
+            if self.data_processor.get_value_from_tree("Track_cand_index_"+AKx) is not None:
+                track_cand_index = self.data_processor.get_value_from_tree("Track_cand_index_"+AKx, self.i_event)
```

Comment:
Same with this check here.

Reply:
This one can can certainly be removed, I'll check that.